### PR TITLE
Homework 7

### PR DESCRIPTION
**Pull request by danipavic**
This PR addresses Homework 7 assignments.

https://github.com/user-attachments/assets/c8d44f15-56a4-4cce-96f3-34eeeb697344

### CODE REVIEW

File: tv-shows/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	"version": "0.1.0",
 	"private": true,
 	"scripts": {
-		"dev": "next dev",
+		"dev": "next dev -p 4200",
```
Comment (darrac) [inline tv-shows/package.json:6]: Just like the good old times

File: tv-shows/src/components/features/shows/ShowPicker/components/ShowPickerStep.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+			);
+
+			setSelectedShows([...filteredShows, show]);
+		}
+	};
+
+	return (
+		<Flex flexDirection={{ base: 'column', md: 'row' }} align="center" justify="space-between" gap="8">
+			{activeStepShows.map((availableShow) => (
+				<ShowCard
```
Comment (darrac) [inline tv-shows/src/components/features/shows/ShowPicker/components/ShowPickerStep.tsx:32]: image
What kind of poopy UI is this?

File: tv-shows/src/components/features/shows/ShowPicker/components/ShowPickerProgress.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+'use client';
+
+import { Progress } from '@chakra-ui/react';
+import { useContext } from 'react';
+import { ShowPickerContext } from './ShowPickerContextProvider';
+
+export const ShowPickerProgress = () => {
```
Comment (darrac) [inline tv-shows/src/components/features/shows/ShowPicker/components/ShowPickerProgress.tsx:7]: image
Progress component tells me I'm done here, but I don't see the results.

File: tv-shows/src/components/features/shows/ShowPicker/components/ShowPickerStep.tsx (hunk 2 above)
Comment (danipavic) [reply tv-shows/src/components/features/shows/ShowPicker/components/ShowPickerStep.tsx:32]: Discussed this on consultations

File: tv-shows/src/components/features/shows/ShowPicker/components/ShowPickerProgress.tsx (hunk 3 above)
Comment (danipavic) [reply tv-shows/src/components/features/shows/ShowPicker/components/ShowPickerProgress.tsx:7]: Offset by one

**Pull request merged**